Content.Shared/Containers/ContainerFillSystem.cs

```diff
@@ -42,4 +46,37 @@ private void OnMapInit(EntityUid uid, ContainerFillComponent component, MapInitE
             }
         }
     }
+
+    private void OnTableMapInit(Entity<EntityTableContainerFillComponent> ent, ref MapInitEvent args)
+    {
+        if (!TryComp(ent, out ContainerManagerComponent? containerComp))
+            return;
+
+        if (TerminatingOrDeleted(ent) || !Exists(ent))
+            return;
+
```

Comment:
I was wondering if this was possible on mapinit but I realised some of the spawners do this shit.